File: starterkits/javascript/Solver.js
const Game = require("./GameInterface");

class Solver {
  constructor() {
    // This method should be use to initialize some variables you will need throughout the challenge.
  }

  /**
   * Here is where the magic happens, for now the answer is a single 'I'. I bet you can do better ;)
   *
   * @param {Game.GameMessage} gameMessage
   * @returns {Game.Answer}
   */
  getAnswer(gameMessage) {
    const question = gameMessage.payload;
    console.log("Received Question: ", JSON.stringify(question));

    const totems = [
      {
        shape: "I",
        coordinates: [
          [0, 0],
          [1, 0],
          [2, 0],
          [3, 0],
        ],
      },
    ];

    const answer = { totems };
    console.log("Sending Answer: ", JSON.stringify(answer));
    return answer;
  }
}

module.exports = { Solver };

File: starterkits/csharp/Solver.cs
using System;
using System.Collections.Generic;
using Newtonsoft.Json;

namespace Blitz2022
{
    public class Solver
    {
        public Solver()
        {
            // This method should be use to initialize some variables you will need throughout the challenge.
        }

        /*
        * Here is where the magic happens, for now the answer is empty. I bet you can do better ;)
        *
        */
        public Answer getAnswer(GameMessage gameMessage)
        {
            Question question = gameMessage.payload;
            Console.WriteLine("Received Question: {0}", JsonConvert.SerializeObject(question));

            IList<TotemAnswer> totems = new List<TotemAnswer> {
                new TotemAnswer {
                    shape = Totem.I,
                    coordinates = new List<CoordinatePair> {
                        (0,0),
                        (1,0),
                        (2,0),
                        (3,0)
                    }
                }
            };

            Answer answer = new Answer
            {
                totems = totems
            };
            Console.WriteLine("Sending Answer: {0}", JsonConvert.SerializeObject(answer));
            return answer;
        }
    }
}